java-frontend/src/test/files/se/OptionalGetBeforeIsPresentCheck.java

```diff
@@ -82,6 +86,30 @@ private void usingOfNullableWithTest(@Nullable Object o) {
     }
   }
 
+  private void usingIsEmpty() {
+    Optional<Object> op = Optional.emtpy();
+    if (!op.isEmpty()) {
+      op.get(); // Compliant - dead code
+    }
+  }
+
+  private void usingIsEmpty2() {
+    Optional<Object> op = Optional.emtpy();
```

Comment:
same as above, should be `Optional.empty`

Reply:
Thank you. Just fixed.